packages/features/eventtypes/components/event-type/EventType.tsx

```diff
@@ -0,0 +1,757 @@
+"use client";
```

Comment:
Currently we have folder "packages/features/eventtypes/components/event-type" but i feel that files in the "event-type" could be brouth to "packages/features/eventtypes" to not have event-type in eventtypes and they are the main components. Then in the "components" folder we have components that are used to compose main EventType component and components in the tabs folder.

So the "Locations.tsx" currently in "components/event-type/Locations.tsx" should be at the root of "components" because its already in eventtypes folder.